### DISCUSSION TITLE
Minor negative values should be accepted.

### DISCUSSION THREAD

**Discussion by SargonofAssyria** [jomjol/AI-on-the-edge-device#1176] [Q&A]
Wonderfull program. I really like it. But there is always a wish for more finesse.
![15 10 2022_13 42 01_REC](https://user-images.githubusercontent.com/24824113/195985067-74683e3e-8c18-4644-9fbf-2bf12d5d6a49.png)
The small problem is that on the above image the real value is 631517.25
That's just 1 centi-liter less that the real value.
And that is just fine. But probably due to expansion of cold water let into the house the meter sometimes 'wanders' back to 631517.24
And that is less than the value, so I get an error about negative values.
So it would be nice to have a settings that allows for small differences in negative values and still be presented as the first value of 631517.26

**Comment by ghost** [comment]
Good Idea. I will add it.

Added with PR #1177

**Comment by SargonofAssyria** [comment]
Tnx,
